### ISSUE TITLE
Invalid parameter flags

### ISSUE THREAD

**Issue by joshuaspence** [jfryman/puppet-nginx#586]
I'm getting this error running on Ubuntu 12.04 with Puppet 2.7.11:

```
Invalid parameter flags at /etc/puppet/ext_modules/nginx/manifests/service.pp:45
```

**Comment by filippomortari** [comment]
It works fine with version 0.2.2.
On my configuration, the problem appeared with version 0.2.4

**Comment by cure** [comment]
I have the same problem on puppet 3.4.3 with version 0.2.4. It worked fine on 0.2.2

**Comment by 3flex** [comment]
This is because #552 was introduced in 0.2.3, breaking things for people on older versions of Puppet that weren't running OpenBSD. The fix (#569) is in master but unreleased on the Forge.

@jfryman I suggest a 0.2.5 release to fix this.

**Comment by jfryman** [comment]
Released 0.2.5 on the Forge
